options/base_options.py

```diff
@@ -31,6 +31,9 @@ def initialize(self, parser):
         parser.add_argument('--ngf', type=int, default=64, help='# of gen filters in the last conv layer')
         parser.add_argument('--ndf', type=int, default=64, help='# of discrim filters in the first conv layer')
         parser.add_argument('--netD', type=str, default='basic', help='specify discriminator architecture [basic | n_layers | pixel]. The basic model is a 70x70 PatchGAN. n_layers allows you to specify the layers in the discriminator')
+        parser.add_argument('--netD2', type=str, default='none', help='specify discriminator architecture [basic | n_layers | pixel]. The basic model is a 70x70 PatchGAN. n_layers allows you to specify the layers in the discriminator')
```

Comment:
So it could be `--netD_global` instead, and thus there's no option as to whether it's basic, n_layers or pixel, right ?

Reply:
Yes if we change model `--netD2` into `netD_global`, it only can be stylegan2 discriminator or torchvision models such as resnet18 for example.